swift/WorkflowUI/Sources/Screen/AnyScreen/AnyScreen.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+public struct AnyScreen: Screen {
+    internal let wrappedScreen: Screen
+    private let viewControllerBuilder: (ViewRegistry) -> AnyScreenViewController.WrappedViewController
+
+    public init<T: Screen>(_ screen: T) {
+        self.wrappedScreen = screen
+        self.viewControllerBuilder = { viewRegistry in
+            return viewRegistry.provideView(for: screen)
+        }
+    }
+
+    // Internal method to inflate the wrapped screen from a view registry
+    func makeViewController(from viewRegistry: ViewRegistry) -> AnyScreenViewController.WrappedViewController {
```

Comment:
With both this and the `AnyScreenViewController` type being internal, how would we go about building containers that can show any screen type?

Right now, it looks like the `ContainerViewController` is the only one that can inflate (since it's within the same module).

Previously, since there was an `inflate` method on `Screen`, the other containers (like backstack, modal, etc) could also handle inflating.

Off the cuff, I would propose adding an inflate method like
```
    public func inflate(from viewRegistry: ViewRegistry) -> ScreenViewController<AnyScreen> {
        return viewRegistry.provideView(for: self)
    }
```

That will still return a `ScreenViewController` for the screen, but I'm open to something else. Just need to have a path for making container that can take `AnyScreen`

Alternatively, could expose the `provideView` on the view registry.

Reply:
`provideView` was supposed to be public – that was an oversight. I just added a commit to make it public.